django/test/runner.py

```diff
@@ -514,6 +535,20 @@ def run(self, result):
     def __iter__(self):
         return iter(self.subsuites)
 
+    def initialize_suite(self):
+        if multiprocessing.get_start_method() == "spawn":
+            self.initial_settings = {
+                alias: connections[alias].settings_dict for alias in connections
+            }
+            self.serialized_contents = {
+                alias: connections[alias]._test_serialized_contents
+                for alias in connections
+                if alias in self.serialized_aliases
+            }
+        else:
+            self.initial_settings = None
+            self.serialized_contents = None
```

Comment:
I would move initialization of these variables to the `ParallelTestSuite.__init_()`.